.github/workflows/test-ui.yml

```diff
@@ -23,6 +23,32 @@ jobs:
         uses: ./.github/actions/setup-vuu-ui
       - run: cd ./vuu-ui && npm run test:vite
 
+  playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install Node & dependencies
+        uses: ./.github/actions/setup-vuu-ui
+      - name: Install dependencies
+        run: npm ci
+      - name: Run Vuu Showcase
+        run: npm run showcase:build && npm run showcase:preview
```

Comment:
I think its quite nice to give the two stepd separate labels, the way the cypress task does
Also, wouldn't it be useful to have the wait here too, to ensire the page is loaded
```
          build: npm run showcase:build
          start: npm run showcase:preview
          wait-on: "http://localhost:4173"
 ```